tests/tensor/test_basic.py

```diff
@@ -941,6 +957,33 @@ def check_u(m, k=0):
             check_u(m, 1)
             check_u(m, -1)
 
+            m = random_of_dtype((10, 10, 10), dtype)
+            check_l_batch(m, 0, tensor3)
```

Comment:
This is a bit costly as every check will compile a new function for a single evaluation. You can reuse the same function to test different values of `k`

Reply:
Can we use a `for` loop to check different value of `k` ? like:
```python
def check_u_batch(m):
    m_symb = tensor3(dtype=m.dtype)
    k_symb = iscalar()
    f = function([m_symb, k_symb], triu(m_symb, k_symb))
    for k in [-1,0,1]:
        result = f(m, k)
        assert np.allclose(result, np.triu(m, k))
        assert result.dtype == np.dtype(dtype)
```

Reply:
sure